transactions/.env

```diff
@@ -0,0 +1 @@
+DATABASE_URL=postgresql://postgres:postgres@localhost:5432/transactions
```

Comment:
Missing new line...
Why even bother asking for an approval ? Do you even care ?